adm_program/modules/dates/dates_function.php

```diff
@@ -575,9 +575,37 @@
     $member = new TableMembers($gDb);
     $member->readDataByColumns(array('mem_rol_id' => $date->getValue('dat_rol_id'), 'mem_usr_id' => $gCurrentUser->getValue('usr_id')));
     $member->setValue('mem_comment', $getUserComment);
-    $member->setValue('mem_count_guests', $getAdditionalGuests);
+    // If participation limit is set
+    if ($date->getValue('dat_max_members') > 0)
+    {
+        // First store possible previous guests assignment and reset value
+        $currentGuests = $member->getValue('mem_count_guests');
+        $member->setValue('mem_count_guests', NULL);
```

Comment:
You are right. This is non sense. This was old code from first implementation before rework.
Deleted it.